docs/functionalabstractions/concretetypes/debug.md

```diff
@@ -3,4 +3,187 @@ id: debug
 title: "Debug"
 ---
 
-Coming soon.
+`Debug[A]` describes the ability to render a value of type `A` to a human readable format for debugging purposes.
```

Comment:
This is good, it contains everything one needs to get started, and it even has some nice examples. Having worked with it, readers might miss out some of the nice capabilities by reading this doc too quick or not thinking hard enough. More substantial and meaningful examples from the community (or a talk) could be quite helpful to make this point across.